aziotctl/aziotctl-common/src/config/apply.rs

```diff
@@ -103,7 +111,20 @@ pub fn run(
 
                     super_config::ManualAuthMethod::X509 { identity } => {
                         match identity {
-                            super_config::X509Identity::Issued { identity_cert } => {
+                            super_config::X509Identity::Issued { mut identity_cert } => {
+                                if let aziot_certd_config::CertIssuanceMethod::Est {
+                                    url: _,
+                                    ref mut auth,
```

Comment:
You don't need the `ref mut`. You're matching a `&mut` so `auth` is already a `&mut` itself.

Reply:
Fixed